cadquery/hull.py

```diff
@@ -385,14 +409,17 @@ def find_hull(edges: Iterable[Edge]) -> Wire:
     current_angle = 0.0
     finished = False
 
+    iterations = 0
+
     # march around
-    while not finished:
+    while not finished and iterations < len(entities):
+        iterations += 1
```

Comment:
While working on this it was very easy for this to loop infinitely. Setting a sensible maximum number of iterations (if we've iterated more than the number of entities, _something_ has gone wrong) seems better than having the system hang in an infinite loop.